rust/src/accuracy/polars/mod.rs

```diff
@@ -0,0 +1,203 @@
+use opendp_derive::bootstrap;
+use polars::{
+    datatypes::{AnyValue, DataType, Field},
+    frame::{row::Row, DataFrame},
+    prelude::{IntoLazy, LazyFrame, Schema},
+};
+use polars_plan::{
+    dsl::{AggExpr, Expr},
+    plans::DslPlan,
+};
+
+use crate::{
+    accuracy::{
+        discrete_gaussian_scale_to_accuracy, discrete_laplacian_scale_to_accuracy,
+        gaussian_scale_to_accuracy, laplacian_scale_to_accuracy,
+    },
+    core::{Measure, Measurement, Metric, MetricSpace},
+    domains::LazyFrameDomain,
+    error::Fallible,
+    measurements::{
+        expr_noise::{match_noise, Distribution, Support},
+        expr_report_noisy_max_gumbel::match_report_noisy_max_gumbel,
+    },
+    polars::{ExtractLazyFrame, OnceFrame},
+    transformations::expr_discrete_quantile_score::match_discrete_quantile_score,
+};
+
+#[cfg(feature = "ffi")]
+mod ffi;
+
+#[bootstrap(
+    name = "onceframe_measurement_utility",
```

Comment:
Generally prefer functions to have verb-y names... maybe something like `describe_measurement`?

Reply:
Updated to `describe_onceframe_measurement_accuracy` (even though it might not return accuracy if you don't pass alpha)